astropy/coordinates/tests/test_iau_fullstack.py

```diff
@@ -187,21 +186,32 @@ def test_against_pyephem():
     https://github.com/phn/pytpm/issues/2#issuecomment-3698679
     """
     obstime = Time('2011-09-18 08:50:00')
-    location = EarthLocation(lon=Angle('-109d24m53.1s'), lat=Angle('33d41m46.0s'),height=30000.*u.m)
+    location = EarthLocation(lon=Angle('-109d24m53.1s'),
+                             lat=Angle('33d41m46.0s'),
+                             height=30000. * u.m)
+    # This is the default pressure and temperature in PyEphem
     temperature = 15 * u.deg_C
     pressure = 1.010 * u.bar
+    # pressure = 0 * u.bar
     # relative_humidity = ?
     # obswl = ?
-    aaframe = AltAz(obstime=obstime,location=location,
-                    temperature=temperature, pressure=pressure)
+    altaz_frame = AltAz(obstime=obstime, location=location,
+                        temperature=temperature, pressure=pressure)
 
-    altaz = SkyCoord('-60.7665d 6.8927d', frame=aaframe)
+    altaz = SkyCoord('6.8927d -60.7665d', frame=altaz_frame)
     radec_actual = altaz.transform_to('icrs')
-    radec_expected = SkyCoord('196.497518d -4.569323d', frame='icrs')
-    distance = radec_actual.separation(radec_expected)
 
-    # TODO: what assertion precision makes sense here?
-    assert distance < 1 * u.arcsec
+    radec_expected = SkyCoord('196.497518d -4.569323d', frame='icrs')  # EPHEM
+    # radec_expected = SkyCoord('196.496220d -4.569390d', frame='icrs')  # HORIZON
+    distance = radec_actual.separation(radec_expected).to('arcsec')
+    # TODO: why is this difference so large?
+    # It currently is: 31.45187984720655 arcsec
```

Comment:
@cdeil - can you make a plot of the offset for many different inputs to see if it is always in the same direction, or always around 31 arcsec? I.e. is it systematic?